src/assets/wise5/common/object/object.ts

```diff
@@ -1,3 +1,7 @@
 export function copy(obj: any): any {
   return typeof obj === 'undefined' ? undefined : JSON.parse(JSON.stringify(obj));
 }
+
+export function serverSaveTimeComparator(object1: any, object2: any): number {
+  return object1.serverSaveTime - object2.serverSaveTime;
+}
```

Comment:
I think adding type to the input object params would make this function input more clear.
```suggestion
export function serverSaveTimeComparator(
  object1: { serverSaveTime: number },
  object2: { serverSaveTime: number }
): number {
  return object1.serverSaveTime - object2.serverSaveTime;
}
```

Reply:
Yep, makes sense.